cmd/getPromoter/getPromoter.go

```diff
@@ -0,0 +1,57 @@
+// Command Group: "GTF Tools"
+
+// getPromoter will take a list of unique genes of interest and return a bed file of the promoter region proceeding
+// the position of the TSS for each isoform by the amount specified with upstream and following the TSS by the amount specified by downstream.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/chromInfo"
+	"github.com/vertgenlab/gonomics/fileio"
+	gtf2 "github.com/vertgenlab/gonomics/gtf"
+	"log"
+)
+
+func getPromoter(genes string, info string, outBed string, chrom string, upstream int, downstream int) {
+	geneNames := fileio.Read(genes)
+	geneInfo := gtf2.Read(info)
+	chr := chromInfo.ReadToMap(chrom)
+
+	answer := gtf2.FindPromoter(geneNames, upstream, downstream, geneInfo, chr)
+
+	bed.Write(outBed, answer)
+}
+
+func usage() {
+	fmt.Print(
+		"getPromoter will take a list of unique genes of interest and return a bed file of the promoter region proceeding\n" +
+			"the position of the TSS for each isoform by the amount specified with upstream and the amount following the TSS as specified by downstream. This program is strand aware.\n" +
+			"Usage:\n" +
+			"getPromoter [options] uniqueGenes.txt geneInfo.gtf out.bed chrom.sizes\n" +
```

Comment:
Would it be helpful to specify in the lines above that geneInfo.gtf and chrom.sizes need to be provided as input?